app/src/main/java/com/DatabaseHandler.java

```diff
@@ -0,0 +1,122 @@
+package com;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.DatabaseUtils;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.util.Log;
+
+import com.simpragma.recipe.recipeapplication1.DataBaseResult;
+import com.simpragma.recipe.recipeapplication1.Result;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by shubhank on 2/4/18.
+ */
+
+public class DatabaseHandler extends SQLiteOpenHelper {
+
+    // All Static variables
+    // Database Version
+    private static final int DATABASE_VERSION = 2;
+
+    // Database Name
+    private static final String DATABASE_NAME = "recipeManager1";
```

Comment:
Why there is 1 in all names?

Reply:
Modified  recipeManager1 as recipeManager